widgets/refinement-list.js

```diff
@@ -69,7 +71,7 @@ function refinementList({
   }
 
   if (templates !== defaultTemplates) {
-    templates = Object.assign({}, defaultTemplates, templates);
+    templates = defaults({}, templates, defaultTemplates);
```

Comment:
Object.assign needs another polyfill everywhere BUT chrome latest. So I would have add to add another polyfill that would just have duplicated some lodash code.

Again a win for lodash here :)